content/en/notebooks/notebooks_new.md

```diff
@@ -0,0 +1,351 @@
+
+- **Editing**: Make changes to the notebook.
+- **Viewing**: Contents are Read Only, preventing users from making unwanted edits to existing configurations and information.
+
+### Version history
+
+From a notebook, click the **Configure** icon and click **Version history** to open the Version History side panel. You can preview, restore, or clone your notebook's version history. For more information, see the [Version History guide][18].
+
+### Graph snapshots
+
+Notebooks can be set to automatically take snapshots of graphs that might expire. Enable this by clicking **Turn on snapshots** in the cog menu of any notebook. Use the cog menu to view snapshots or turn off automatic snapshots. Turn off automatic snapshots to remove access to existing snapshots.
+
+{{< img src="notebooks/cog_snapshots.png" alt="Cog menu option to turn on snapshots" style="width:100%;">}}
+
+ Notebooks with snapshots enabled automatically capture a static image of any graphs with a fixed time range (for example, `Aug 18, 12:00 am - Aug 19, 11:59 pm`). These snapshots update when the graph is updated, as long as the new graph also has a fixed time range. Changing the graph to a global time range (like `Past 1 Hour`) removes the snapshot.
+
+ You can preview the existing snapshot on any fixed-time graph by hovering over the camera icon while in edit mode.
+
+To share a version of your notebook with snapshots, from the cog menu, click **View snapshots**. Copy the URL, or append `&view=snapshots` to the URL of any notebook that has snapshots enabled.
+
+### Permissions
+
+By default, all users have full access to notebooks.
+
+Use granular access controls to limit the [roles][19] that may edit a particular notebook:
+1. While viewing a notebook, click on the cog in the upper right. The settings menu opens.
+1. Select **Permissions**.
+1. Click **Restrict Access**.
+1. The dialog box updates to show that members of your organization have **Viewer** access by default.
+1. Use the dropdown to select one or more roles, teams, or users that may edit the notebook.
+1. Click **Add**.
+1. The dialog box updates to show that the role you selected has the **Editor** permission.
+1. Click **Save**
+
+**Note:** To maintain your edit access to the notebook, the system requires you to include at least one role that you are a member of before saving. 
+
+To restore general access to a notebook with restricted access, follow the steps below:
+1. While viewing the notebook, click on the cog in the upper right. The settings menu opens.
+1. Select **Permissions**.
+1. Click **Restore Full Access**.
+1. Click **Save**.
+
+## Finding notebooks
+
+The [Notebooks List][1] page is the place to find all of your Notebooks.
+
+<!-- TODO Add updated image from new notebooks -->
+
+### Search
+
+The search field supports document text search. Type what you're looking for and the notebooks below will list the results.
```

Comment:
As above, is a document a different thing here?

Reply:
```suggestion
The search field supports text search. Type your query to display the relevant notebooks as results.
```